packages/cozy-stack-client/src/FileCollection.js

```diff
@@ -518,7 +518,9 @@ class FileCollection extends DocumentCollection {
   }
 
   /**
-   * async updateFileMetadata - Updates a file's metadata
+   * async updateFileMetadata - Updates a file / folder's attributes except
+   * the metadata attribute. To update its metadata use `updateFileMetadataAttribute`
+   * see https://docs.cozy.io/en/cozy-doctypes/docs/io.cozy.files/#folder
```

Comment:
`updateFileMetada` does not update `metadata` ? IMHO this is surprising and prone to errors. Do you think we could find names that are less prone to errors ? Also, I do not really understand the usecases here, do you think we could add to the docstrings when we want to do one but not the other ?

Reply:
Also, the link is for "folder" but the method is for a file, is it wanted ?

Reply:
updateFileMetadata should be called `updateAttributes` but even with this name we have : 
```
io.cozy.files: 
data : {
   attributes: {
     id: 'cc',
    name:'ttt',
   metadata: {}
}
}
```

So the explicit name should be : `updateAttributeAndNotMetatada()`... 

`folder` or `file` is the "same thing". It's an io.cozy.files, but folder and files have differents attributes. Like a `file` as a `size` that `folder` doesn't have. 

what do you mean by `doctrings`?

Reply:
TBC, if you want to update the title attribute you use the older method. If you and to update something stored in the `metadata` attribute, then you have to use `updateFileMetadataAttribute`.

Reply:
> folder or file is the "same thing"

For the user of these methods, it is not, we should not leak this abstraction into our docs IMHO. As a user of the doc, clicking on a method of a file and landing on the folder is weird :/

> what do you mean by doctrings?

docstrings = doc des fonctions, c est un terme qui vient de python (où les docstrings sont supportés directement par le langage, tu peux acceder a la doc d'une fonction via `maFonction.__doc__`)

Reply:
I still think that the names are error prones, it would be easier if you could include usecases directly in the examples of the doc.